src/main/java/edu/texas/social_computing/hospitals/HRPP.java

```diff
@@ -1,22 +1,47 @@
 package edu.texas.social_computing.hospitals;
 
-public class HRPP {
-    // run hospital-resident matching alg (couple agnostic)
-
-    // check all couples for proximity violations (location mismatch)
-
-    // while (couple proximity violations exist)
-    // Task 1
-
-        // pick non-dominant partner (not matched partner or partner with worse preference match)
+import com.google.common.collect.ImmutableList;
 
-        // remove all hospitals violating proximity constraint (make a copy dont modify original preference)
+import java.util.*;
 
-        // put back in the queue
-
-        // try to match that person
-
-        // if matched -> good
+public class HRPP {
+    private ResidentTable residentTable;
+    private HospitalTable hospitalTable;
+    private Matching matching;
```

Comment:
Do you think we need these as fields, or could they just be local variables in `run()`, and then `run()` could be a static method and return a `Matching`?